contracts/bridge/contracts/common/trust/LightClientBSCMock.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity >=0.6.0 <0.7.0;
+pragma experimental ABIEncoderV2;
+
+contract LightClientBSCMock {
+    struct StorageProof {
+        uint256 balance;
+        bytes32 codeHash;
+        uint256 nonce;
+        bytes32 storageHash;
+        bytes accountProof;
+        bytes storageProof;
+    }
+
+    uint256 public immutable OUTBOUND_COMMITMENT_POSITION;
+    uint256 public immutable INBOUND_COMMITMENT_POSITION;
+
+    // bridgedChainPosition => lanePosition => bound
+    mapping(uint256 => mapping(uint256 => address)) inbounds;
+    mapping(uint256 => mapping(uint256 => address)) outbounds;
+
+    constructor(uint256 outbound_commitment_position, uint256 inbound_commitment_position) public {
+        OUTBOUND_COMMITMENT_POSITION = outbound_commitment_position;
+        INBOUND_COMMITMENT_POSITION = inbound_commitment_position;
+    }
+
+    function verify_messages_proof(
+        bytes32 outboundLaneDataHash,
+        bytes32 /*inboundLaneDataHash*/,
```

Comment:
Why we need this inboundLaneDataHash?

Reply:
To unify the interface of two different light client proofs.
For beefy light client, it uses `laneDataHash (OutboundDataHash, InboundDataHash)` for message commitment proof, 
and for bsc light client, it uses only `OutboundDataHash` or `InboundDataHash` for storage proof.
```
interface ILightClientBridge {
    function verify_messages_proof(
        bytes32 outboundLaneDataHash,
        bytes32 inboundLaneDataHash,
        uint256 chain_pos,
        uint256 lane_pos,
        bytes calldata proof
    ) external view returns (bool);

    function verify_messages_delivery_proof(
        bytes32 outboundLaneDataHash,
        bytes32 inboundLaneDataHash,
        uint256 chain_pos,
        uint256 lane_pos,
        bytes calldata proof
    ) external view returns (bool);
}
```